idb/postgres/internal/writer/writer_test.go

```diff
@@ -1293,57 +1293,6 @@ func TestWriterAccountAppTableCreateDeleteSameRound(t *testing.T) {
 	assert.Equal(t, block.Round(), basics.Round(closedAt))
 }
 
-// Check that adding same block twice does not result in an error.
```

Comment:
Seems like we should still want to avoid an error in this case. Why remove the test?

Reply:
COPY doesn't allow overriding rows. This test was needed because with the old indexer imported blocks could be ahead of accounting.

Reply:
I think you'd still have that case if there were multiple writers deployed

Reply:
1. No, because everything is done in one transaction.
2. We don't support multiple writers.

Reply:
Do we have a test like this:
```diff
- // Check that adding same block twice does not result in an error.
+ // Check that adding same block twice results in an error.
```

Reply:
There is `TestMultipleWriters()` in `postgres_integration_test.go`.